logging/src/main/java/alluxio/logging/shell/FsMkdirFlowLoggingAspect.java

```diff
@@ -0,0 +1,42 @@
+package alluxio.logging.shell;
+
+import alluxio.logging.shell.base.FsFlowLoggingAspect;
+import org.aspectj.lang.ProceedingJoinPoint;
+import org.aspectj.lang.annotation.Around;
+import org.aspectj.lang.annotation.Aspect;
+
+/**
+ * This class represents a {@link alluxio.logging.base.FlowLoggingAspect} for the
+ * file system command {@code mkdir}. All methods triggered by this command are logged
+ * and kept track of.
+ */
+@Aspect
+public final class FsMkdirFlowLoggingAspect extends FsFlowLoggingAspect {
+
+    private static final String FLOW_NAME = "FsMkdirFlow";
+    private static final String START_METHOD = "execution(* alluxio.cli.fs.command.MkdirCommand.run(..)) || "
+            + "execution(* alluxio.client.file.createDirectory(..))";
+
+    @Around(START_METHOD)
+    @Override
+    public Object startFlux(final ProceedingJoinPoint point) throws Throwable {
```

Comment:
Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.